model/godleyTab.h

```diff
@@ -28,7 +28,9 @@ namespace minsky
   class GodleyTab: public ItemTab
   {	  
   public:
-    bool itemSelector(ItemPtr i) override {if (dynamic_cast<GodleyIcon*>(i.get())) return i.get(); return false;}
+    bool itemSelector(ItemPtr i) override {if (i->classType()=="GodleyIcon") return true; return false;}
```

Comment:
Wouldn't return dynamic_cast<GodleyIcon*>(i.get())) be better?

Reply:
Reverted to the dynamic_cast, thanks :)